### DISCUSSION TITLE
Allow activating a new blueprint version while game server is online

### DISCUSSION THREAD

**Discussion by mixels** [gameserverapp/Platform#976] [Suggestions]
Hi!

Requiring a game server to be offline to change the blueprint version is disruptive because it prevents me from making minor changes unless I fully shut the server down to manually activate the new blueprint version on relevant game servers.

I would please like to request a way to schedule an update of game server blueprint such that the game server will automatically adopt the specified version when it next restarts. This would let me take advantage of scheduled restarts to prevent disruption to server uptime when changing something like a backup path or non-breaking change to command line flags.

**Comment by mvaessen** [comment]
Thanks for the suggestion.
Unfortunately is this not a thing we can do, because it is likely to cause breaking issues.
